Source/Fuse.Controls.Native/Android/SystemScreenConfig.uno

```diff
@@ -0,0 +1,175 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+using Fuse.Controls.Native;
+
+namespace Fuse.Controls
+{
+	/**
+		TODO write docs please
+	*/
+	public extern(Android) class SystemScreenConfig : SystemScreenConfigBase
```

Comment:
Don't mark `SystemScreenConfig` as `extern(Android)`, instead create an internal helper-class that has `extern(Android)` instead.

This is because otherwise, the API actually changes based on the platform. We would rather just have the implementation change.